### DISCUSSION TITLE
Winlogbeat forwarding options

### DISCUSSION THREAD

**Discussion by PatRussell87** [Security-Onion-Solutions/securityonion#2177] [Unsupported Versions] [answered]
Is the management node the only destination for Windows event log forwarding? I have a protected environment that I would like to centrally monitor but I would prefer not to have every endpoint forwarding out of it. I was hoping to have some solution that would allow me to aggregate the logs and make them available from one endpoint. Would a heavy node accomplish this?

**Comment by defensivedepth** [comment, marked as answer]
By default, you can ship Windows Eventlogs with Winlogbeat, Osquery or Wazuh. 

Winlogbeat would need to be installed on each endpoint and pointed at the Manager.
Osquery would be need to be installed on each endpoint and pointed at the Manager or a Fleet Standalone Node.
Wazuh would be need to be installed on each endpoint and pointed at any type of Node.

Alternatively, you could aggregate all the endpoint logs to a single endpoint inside the protected enclave (using something like Windows Event Forwarding), and then use Winlogbeat/Osquery/Wazuh on that single endpoint to ship logs out to Security Onion.

**Reply by sleepingbel** [reply]
For windows eventlog's you can use a Windows Event Collector (WEC) and than install only on the WEC winlogbeat. This is the solution that we use and it works. 

You need to make a GPO that the end-user device forwards its logs to the WEC, and so you do not have to install agents on the end-user devices.

**Comment by PatRussell87** [comment]
Thank you both for the tips. Gives me something to look at as I plan my upgrade from 16 to 2.x
